src/app/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.ts

```diff
@@ -102,6 +105,25 @@ export class UbsAdminTableComponent implements OnInit, OnDestroy {
     }
   }
 
+  getColumns() {
+    this.adminTableService
+      .getColumns()
+      .pipe(takeUntil(this.destroy))
+      .subscribe((columns: any) => {
+        this.columns = columns.columnStateDTOList;
+        this.columns.forEach((column, index) => {
+          this.displayedColumns[index] = column.title.key;
+        });
+        this.arrayOfHeaders = this.columns.filter((item, index) => index !== 0);
+        this.orderInfo = this.arrayOfHeaders.slice(0, 3);
+        this.customerInfo = this.arrayOfHeaders.slice(3, 10);
+        this.orderDetails = this.arrayOfHeaders.slice(10, 18);
+        this.sertificate = this.arrayOfHeaders.slice(18, 22);
+        this.detailsOfExport = this.arrayOfHeaders.slice(22, 27);
+        this.responsiblePerson = this.arrayOfHeaders.slice(27, 33);
```

Comment:
Can we improve this code?
Looks very strange

Reply:
I know I saved it so as not to break the code, it will be changed in the near future.